templates/paragraphs/paragraph--oe-facts-figures.html.twig

```diff
@@ -7,9 +7,11 @@
  */
 #}
 {% if title is not empty %}
-  <h2 class="ecl-u-type-heading-2">
-    {{ title }}
-  </h2>
+  <div class="ecl">
+    <h2 class="ecl-u-type-heading-2">
```

Comment:
As I understand .ecl class was dedicated to automatic styling tags (like h2) in the case when we don't have ecl-related classes. Like HTML coming from ckeditor.
Is it from ecl team recommendation?

Reply:
It is not a recommendation from the ECL team but yes, the "ecl" was added to style the heading because the margin of the heading:
`heading-margin: var(--s-2xl) 0 var(--s-m)` - 2xl top and m bottom does not apply if the wrapper does not have the "ecl" class.